bundles/org.openhab.binding.openwebnet/README.md

```diff
@@ -278,10 +278,10 @@ _Notes on `shutter` position_
 Basic Scenarios and CEN/CEN+ Scenarios channels are [TRIGGER channels](https://www.openhab.org/docs/configuration/rules-dsl.html#channel-based-triggers]): they handle events and do not have a state.
 
 A powerful feature is to detect scenario activations and CEN/CEN+ buttons press events to trigger rules in openHAB: this way openHAB becomes a very powerful scenario manager activated by BTicino scenario control modules or by CEN/CEN+ scenarios physical buttons.
-See [openwebnet.rules](#openwebnetrules) for examples on how to define rules that trigger on scenarios and on CEN/CEN+ button press events.
+See [openwebnet.rules](#openwebnet-rules) for examples on how to define rules that trigger on scenarios and on CEN/CEN+ button press events.
```

Comment:
Why do you change that ? The link is now apparently no more working.

Reply:
Eclipse .md editor detects the link as wrong and suggests the modified version. Will check which one is correct

Reply:
the links have been restored to previous format